tests/shipit/ShellTest.php

```diff
@@ -25,7 +25,7 @@ abstract class ShellTest extends \Facebook\HackTest\HackTest { // @oss-enable
 
   <<__Override>> // @oss-enable
   public async function afterEachTestAsync(): Awaitable<void> { // @oss-enable
-    await $this->tearDown(); // @oss-enable
+    await $this->tearDownAsync(); // @oss-enable
```

Comment:
Oof, the whitespace on this line doesn't reverse correctly with the commenting logic :/

Reply:
I'm happy to take a crack at fixing the commenting filter, but I could use some pointers/it might not be possible outside FB infra.

For this PR, could you just manually edit the diff?

Reply:
It might not necessarily be a bug. Right now it reverses to:
```
// @oss-enable: function { 
  // @oss-enable: line
// @oss-enable: }
```
but the internal content was:
```
// @oss-enable: function { 
// @oss-enable:   line
// @oss-enable: }
```
I just updated the internal file to match what the logic currently reverses to.

Reply:
I may be missing something but the patch doesn't change the spacing on that line? Shouldn't it be cleanly reversible?

Maybe it's just a consequence of the way the filter rewrites lines but `hackfmt` corrects a bunch of oss-enable/disable lines that I assumed to be a bug.

E.g. `hackfmt -i tests/shipit/FilterSanityCheckPhaseTest.php`:
```diff
diff --git a/tests/shipit/FilterSanityCheckPhaseTest.php b/tests/shipit/FilterSanityCheckPhaseTest.php
index df99368..bec7417 100644
--- a/tests/shipit/FilterSanityCheckPhaseTest.php
+++ b/tests/shipit/FilterSanityCheckPhaseTest.php
@@ -57,7 +57,7 @@ final class FilterSanityCheckPhaseTest extends BaseTest {
       $phase->assertValid(keyset['foo/']);
     })
       // @oss-disable: ->toThrow(\InvariantViolationException::class);
-    ->toThrow(InvariantException::class); // @oss-enable
+      ->toThrow(InvariantException::class); // @oss-enable
   }
 
   public function testThrowsForEmptyChangeset(): void {
@@ -68,7 +68,7 @@ final class FilterSanityCheckPhaseTest extends BaseTest {
       $phase->assertValid(keyset['foo/']);
     })
       // @oss-disable: ->toThrow(\InvariantViolationException::class);
-    ->toThrow(InvariantException::class); // @oss-enable
+      ->toThrow(InvariantException::class); // @oss-enable
   }
 
   public function testThrowsForPartialMatch(): void {
@@ -84,7 +84,7 @@ final class FilterSanityCheckPhaseTest extends BaseTest {
       $phase->assertValid(keyset['foo/', 'herp/']);
     })
       // @oss-disable: ->toThrow(\InvariantViolationException::class);
-    ->toThrow(InvariantException::class); // @oss-enable
+      ->toThrow(InvariantException::class); // @oss-enable
   }
 
   public function testAllowsForStrippedPaths(): void {
```